analyzers/FacsCalibur/nbproject/project.properties

```diff
@@ -0,0 +1,77 @@
+annotation.processing.enabled=true

```

Comment:
This file should not be added to version control. It belongs with the IDE (Netbeans)